.gitignore

```diff
@@ -2,3 +2,4 @@ build
 .idea
 .gradle
 out
+data/
```

Comment:
Doesn't this need to exist with data to load into H2? I don't see where the H2 data comes from.